setup.py

```diff
@@ -30,8 +30,16 @@
 
         license=LICENSE,
 
-        packages=['resolwe_bio'],
-        include_package_data=True,
+        packages=find_packages(),
+        package_data={
+            'resolwe_bio': [
+                'descriptors/*.yml',
+                'processes/**/*.yml',
+                'tests/files/*',
+                'tools/*.py',
+                'tools/*.R',
+            ]
+        },
         zip_safe=False,
         install_requires=(
             "resolwe>=1.0.0",
```

Comment:
So, the general rule is that text printed to the user should be in double quotes and the other in single. You can fix the whole `setup.py` if you wish, but not urgent.